src/main/java/net/authorize/sample/PaymentTransactions/CreateChasePayTransaction.java

```diff
@@ -0,0 +1,108 @@
+package net.authorize.sample.PaymentTransactions;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+
+import net.authorize.Environment;
+import net.authorize.api.contract.v1.ANetApiResponse;
+import net.authorize.api.contract.v1.CreateTransactionRequest;
+import net.authorize.api.contract.v1.CreateTransactionResponse;
+import net.authorize.api.contract.v1.CreditCardType;
+import net.authorize.api.contract.v1.MerchantAuthenticationType;
+import net.authorize.api.contract.v1.MessageTypeEnum;
+import net.authorize.api.contract.v1.PaymentType;
+import net.authorize.api.contract.v1.TransactionRequestType;
+import net.authorize.api.contract.v1.TransactionResponse;
+import net.authorize.api.contract.v1.TransactionTypeEnum;
+import net.authorize.api.controller.CreateTransactionController;
+import net.authorize.api.controller.base.ApiOperationBase;
+
+public class CreateChasePayTransaction {
+
+    //
+    // Run this sample from command line with:
+    //                 java -jar target/ChargeCreditCard-jar-with-dependencies.jar
+    //
+    public static ANetApiResponse run(String apiLoginId, String transactionKey, Double amount) {
+
+
+        //Common code to set for all requests
+        ApiOperationBase.setEnvironment(Environment.SANDBOX);
+
+        MerchantAuthenticationType merchantAuthenticationType  = new MerchantAuthenticationType() ;
+        merchantAuthenticationType.setName(apiLoginId);
+        merchantAuthenticationType.setTransactionKey(transactionKey);
+        ApiOperationBase.setMerchantAuthentication(merchantAuthenticationType);
+
+        // Populate the payment data
+        PaymentType paymentType = new PaymentType();
+        CreditCardType creditCard = new CreditCardType();
+        creditCard.setCardNumber("4111111111111111");
+        creditCard.setExpirationDate("2020-12");
+        // Set the token specific info
+        creditCard.setIsPaymentToken(true);
+        creditCard.setCryptogram("EjRWeJASNFZ4kBI0VniQEjRWeJA=");
+        creditCard.setTokenRequestorName("CHASE_PAY");
+        creditCard.setTokenRequestorId("12345678901");
+        creditCard.setTokenRequestorEci("07");
+        creditCard.setCardCode("999");
+        paymentType.setCreditCard(creditCard);
+
+        // Create the payment transaction request
+        TransactionRequestType txnRequest = new TransactionRequestType();
+        txnRequest.setTransactionType(TransactionTypeEnum.AUTH_CAPTURE_TRANSACTION.value());
+        txnRequest.setPayment(paymentType);
+        txnRequest.setAmount(new BigDecimal(amount).setScale(2, RoundingMode.CEILING));
+        //txnRequest.s
+        // Make the API Request
+        CreateTransactionRequest apiRequest = new CreateTransactionRequest();
+        apiRequest.setTransactionRequest(txnRequest);
+        //apiRequest.setRefId("123456");
```

Comment:
Remove this line

Reply:
I have removed the line that is mentioned and I have pushed as well.